src/main/java/nl/cherement/jak/config/SecurityConstants.java

```diff
@@ -0,0 +1,12 @@
+package nl.cherement.jak.config;
+
+public class SecurityConstants {
+
+    private SecurityConstants(){}
+
+    public static final String SECRET = "JjXBGwNhxo1qTGPIZvpZoOXNBJ664NhPgUGaOoHpVI1X35QVs6";
```

Comment:
Ik zou het dus toch wel nice vinden als je deze in de application.properties stopt voor we hem mergen. Het hebben van een secret maakt niet veel uit als iedereen hem kan zien.

Reply:
Dit si gedaan inmiddels. Maar ik heb geen recthten op travis om dit te encrypten